charts/apbs-rest/templates/id-ingress.yaml

```diff
@@ -12,9 +12,9 @@ metadata:
     heritage: {{ .Release.Service }}
     storage: id
   annotations:
-    nginx.ingress.kubernetes.io/rewrite-target: /api/uid/
+    #nginx.ingress.kubernetes.io/rewrite-target: /api/uid/
     # TODO: 2020/07/02, Elvis - commenting out line below for now since user doesn't need access beyond retrieving a new ID
-    # nginx.ingress.kubernetes.io/rewrite-target: /api/uid/$2
```

Comment:
I'm not sure whether we need to have the rewrite target changed since the validation path only needs to be accessed in-cluster.  Having it accessible from the outside works great for testing however.